qiskit/pulse/utils.py

```diff
@@ -41,3 +46,53 @@ def pad(schedule, channels=None, until=None):
     """
     warnings.warn("The function `pad` has been moved to qiskit.pulse.reschedule. It cannot be "
                   "invoked from `utils` anymore (this call returns None).")
+
+
+def measure(qubits: List[int],
+            schedule: Schedule,
```

Comment:
I was thinking about this a bit. I think it would be better without the schedule input arg. It can be initialized on line 73. This is the difference:
```
sched += measure(qubits, backend)
```
vs
```
sched = measure(qubits), sched, backend)
```

Reply:
You might want to initialize like:
`schedule = Schedule(name="Default measurement schedule for qubits {}".format(qubits))`